src/lib/components/workflow/workflows-summary-configurable-table/table-empty-state.svelte

```diff
@@ -0,0 +1,83 @@
+<script>
+  import { page } from '$app/stores';
+  import Icon from '$lib/holocene/icon/icon.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import { workflowError } from '$lib/stores/workflows';
+  import emptyImage from '$lib/vendor/empty-state-dark_2x.png';
+  import noResultsImages from '$lib/vendor/empty-state-light_2x.png';
+
+  $: query = $page.url.searchParams.get('query');
+
+  const samples = [
+    'samples-go',
+    'samples-java',
+    'samples-typescript',
+    'samples-python',
+    'samples-dotnet',
+    'samples-php',
+  ];
+</script>
+
+<div
+  class="flex flex-col xl:flex-row w-full h-auto rounded-bl-xl rounded-br-xl border-2 border-gray-900 overflow-hidden"
+>
+  <div
+    class="w-auto min-w-[280px] xl:min-w-[520px] p-8 flex flex-col gap-4 bg-white border-b-2 xl:border-b-0 xl:border-r-2 border-gray-900"
+  >
+    <h3 class="text-lg font-medium">
+      {#if query}
+        {translate('workflows', 'workflow-query-empty-state-title')}
+      {:else}
+        {translate('workflows', 'workflow-empty-state-title')}
+      {/if}
+    </h3>
+    {#if $workflowError}
+      <p>
+        {translate('workflows', 'workflow-query-error-state')}
+      </p>
+      <p
+        class="rounded-md border-2 border-orange-500 bg-orange-100 p-5 text-center"
+      >
+        {$workflowError}
+      </p>
+    {:else if query}
+      <p>
+        {translate('workflows', 'workflow-query-empty-state-preface')}
+      </p>
+      <p>
+        {translate('workflows', 'workflow-query-empty-state-postface')}
+      </p>
+    {:else}
+      <slot name="cloud" />
+      <p>
+        {translate('workflows', 'workflow-empty-state-description')}
+        <a
+          href="https://github.com/temporalio"
+          rel="noreferrer"
+          target="_blank"
+          class="underline hover:text-blue-700 hover:decoration-blue-700"
+          >Temporal's GitHub repository</a
+        >.
+      </p>
+      <ul class="flex flex-col gap-2">
+        {#each samples as sample}
+          <li>
+            <a
+              class="flex gap-2 items-center hover:text-blue-700 hover:underline hover:decoration-blue-700"
+              href="https://github.com/temporalio/{sample}"
+              rel="noreferrer"
+              target="_blank"><Icon name="github" />{sample}</a
+            >
+          </li>
+        {/each}
+      </ul>
+    {/if}
+  </div>
+  <div>
+    <img
+      src={query ? noResultsImages : emptyImage}
+      alt=""
+      class="aspect-auto"
```

Comment:
This works great for every scenario, expect for this one 😞 

<img width="1151" alt="Screenshot 2023-08-09 at 3 34 40 PM" src="https://github.com/temporalio/ui/assets/15069288/4dae36e1-4049-4636-83a2-412099108c40">

Two potential solutions might be using the `2xl` breakpoint or using a `flex-wrap` on the samples to try and take up less height on the left 🤔

Reply:
Sometimes the easiest solution is the best. Just made the bg color the same as the mountains.

<img width="1677" alt="Screen Shot 2023-08-10 at 8 47 51 AM" src="https://github.com/temporalio/ui/assets/7967403/695e75b5-35fb-4756-9562-592719167841">
<img width="1293" alt="Screen Shot 2023-08-10 at 8 48 11 AM" src="https://github.com/temporalio/ui/assets/7967403/3b911216-ddaf-4d41-8c0e-f81550b15585">

Reply:
Amazing 🎉